NEWS.md

```diff
@@ -1,60 +1,46 @@
 # xportr (development version)
```

Comment:
```suggestion
# xportr 0.4.0
```

Reply:
I believe you can use https://usethis.r-lib.org/reference/use_version.html to update the news and description file for the release.